tests/test_field_statistics.py

```diff
@@ -892,18 +891,15 @@ def test_main(tmp_path, config_file, config_dict, print_info=True):
     result_path = ens_path / RESULT_PATH
 
     rms_load_script = result_path / RMS_LOAD_SCRIPT_NAME
-    # Run the main script as a subprocess
-    script_name = Path(__file__).absolute().parent.parent / Path(
-        "src/subscript/field_statistics/field_statistics.py"
-    )
-    if print_info:
-        print(f"\nRun script:  {script_name}")
+
+    # Run the main script as a subprocess,but first clean up
+    # result directory to be sure this test does not re-use
+    # previous results from previous tests
     remove_file_path = result_path / Path("ertbox--*.roff")
```

Comment:
I don't think this is needed. `tmp_path` will always be a new path every time the test is initiated.

Reply:
OK, I can remove the subprocess to clean up then.